src/app/shared/related-items/related-items.component.html

```diff
@@ -1,7 +1,8 @@
-<div [hidden]="totalHits == 0">
+<div [hidden]="totalHits == 0 && relationship_type != 'project-specimen' && relationship_type != 'project-organism'">
```

Comment:
Check what is the difference between '==' and '===' in JS and which one we need to use here